# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ldap2dns/Makefile-LIBS.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ldap2dns-0.3.6/Makefile.orig	2006-04-30 17:53:50.000000000 +0200
+++ ldap2dns-0.3.6/Makefile	2006-04-30 17:55:11.000000000 +0200
@@ -30,14 +30,14 @@
 
 debug: ldap2dns-dbg
 
-ldap2dns: ldap2dns.o $(LIBS) 
-	$(LD) $(LDFLAGS) -o $@ $+
+ldap2dns: ldap2dns.o
+	$(LD) $(LDFLAGS) -o $@ $+ $(LIBS)
 
 ldap2dnsd: ldap2dns
 	ln -f ldap2dns ldap2dnsd
 
-ldap2dns-dbg: ldap2dns.o-dbg $(LIBS) 
-	$(LD) $(LDFLAGS) -o $@ $+
+ldap2dns-dbg: ldap2dns.o-dbg
+	$(LD) $(LDFLAGS) -o $@ $+ $(LIBS)
 
 ldap2dns.o: ldap2dns.c
 	$(CC) $(CFLAGS) -c $< -o $@
